streams/src/main/java/org/apache/kafka/streams/kstream/ValueTransformerSupplier.java

```diff
@@ -18,9 +18,14 @@
 package org.apache.kafka.streams.kstream;
 
 /**
- * A value transformer supplier which can create one or more {@link ValueTransformer} instances.
+ * A {@link ValueTransformerSupplier} interface which can create one or more {@link ValueTransformer} instances.
```

Comment:
Question (/cc @guozhangwang):  What determines/influences whether it would create one instance or multiple instanes of `ValueTransformer`?  Is about whether calling `get()` multiple times would always return the same `ValueTransformer` instance vs. new instances on each invocation?  I am asking because there's no parameter to `get()` that would allow the user to make context-aware decisions on which specific instance/behavior of `ValueTransformer` would need to be returned.